api/src/main/kotlin/nl/avisi/anowalke/services/VertaalService.kt

```diff
@@ -0,0 +1,83 @@
+package nl.avisi.anowalke.services
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import mu.KotlinLogging
+import nl.avisi.anowalke.dto.ExpressionsDto
+import nl.avisi.anowalke.dto.Translations
+import nl.avisi.anowalke.util.configureObjectMapper
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.ResponseEntity
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter
+import org.springframework.stereotype.Component
+import org.springframework.web.client.HttpStatusCodeException
+import org.springframework.web.client.RestTemplate
+
+@Component
+class VertaalService {
+
+    val dummylist = listOf("subscription","key","region","endpoint","url","client","inputlist","request","body","media","type","gson","response","list","of","my","class","object","translations","it","post","json_text","parser","json","prettify","translator")
+
+    private val subscriptionKey = "f2a9a9f29d7d4e878669cdfed04f99ce"
+    private val subscriptionRegion = "westeurope"
+    private val endpoint = "https://api-eur.cognitive.microsofttranslator.com/"
+    var url = "$endpoint/translate?api-version=3.0&from=nl&to=en"
+
+    private val restTemplate = RestTemplate().configureJsonMappers()
```

Comment:
Je kan overwegen om dit ergens als bean te definiëren mits het interessant is om dit meerdere keren te gebruiken.